llvm/tools/sycl-post-link/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+find_package(LLVMGenXIntrinsics QUIET)
```

Comment:
Why do we need this call here? And why don't we need the rest of the fallback code to download `vc-intrinsics`?

Reply:
Sorry good catch. I added this to fix some linking errors I saw locally but it seems that was actually a problem with my system config and not the CMake setup. Same story for `sycl-jit`. Will remove both.


FYI @jopperm